docs/_articles/en/lwc/lwc-get-started.md

```diff
@@ -0,0 +1,100 @@
+---
+title: Quick Start LWC Development
+lang: en
+---
+
+## Overview
+You’ll create an LWC component called `contactList` that displays contact names and phone numbers in your App. We’ll get the contact information to display from an Apex class called `ContactController`.
+
+Before you start, install [Salesforce Extensions for Desktop](./en/vscode-desktop/install) or [Set Up Code Builder](./en/codebuilder/cb-setup)
+
+### Add an Apex Class that Queries Contacts
+1. In VS Code, run the command **SFDX: Create Project** to create a Salesforce DX project if you don’t have one.
+2. Log in to the org to which you want to add the LWC.
+3. Run the **SFDX: Refresh SObject Definitions** command from the Command Palette to get completion suggestions for your SObjects related code.
+4. From the command palette run **SFDX: Create Apex Class** and create a class called `ContactController`.
+
+Next let's add a method called `getContacts` to this class. We’ll make this method both public and static. Because it’s a static method, you don't need to create an instance of the class to access the method—you can just use the name of the class followed by a dot (`.`) and the name of the method. This method queries a set of fields on the contact object. Paste this code into `contactController.cls`:
+
+
+```
+public with sharing class ContactController {
+    @AuraEnabled(cacheable=true)
+    public static List<Contact> getContacts() {
+        return [
+SELECT 
+Id, 
+Name, 
+Email, 
+Phone 
+FROM Contact 
+WITH SECURITY_ENFORCED 
+LIMIT 10
+];
+    }
+}
+
+```
+Next we'll add the LWC component that displays queried fields.
+
+### Create an LWC Component that Displays Contact Information
+1. From the command palette run **SFDX: Create Lightning Web Component** and create a component called `contactList` in the default location.
+2. In the `contactList.html file`, cut and paste the following code and save the file:
+
+```
+<template>
+    <lightning-card title="Contact List">
+        <template if:true={contacts.data}>
+            <template for:each={contacts.data} for:item="contact">
+                <div class="slds-var-p-horizontal_medium" key={contact.Id}>
+                    <p>{contact.Name}, {contact.Phone}</p>
+                </div>
+            </template>
+        </template>
+        <template if:true={contacts.error}>
+            <p>{contacts.error}</p>
+        </template>
+    </lightning-card>
+</template>
+
+```
+3. In the contactList.js file, cut and paste this code and save:
+
+```
+import { LightningElement, wire } from 'lwc';
+import getContacts from '@salesforce/apex/ContactController.getContacts';
+
+export default class ContactList extends LightningElement {
+    @wire(getContacts) contacts;
+}
+```
+4. Cut and paste the following code in the contactList.js-meta.xml file and save:
+   
+```
+ <?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I am not sure if there should be a leading space. Could you please confirm that?